docs/Examples.md

```diff
@@ -226,29 +234,37 @@ spec:
   replicas: 3
   storageClassName: local
   varStorage: "128Gi"
+  monitoringConsoleRef:
+    name: example_mc
 EOF
 ```
 
-In the above environment, cluster master controls the [applications loaded](#installing-splunk-apps) to all
-the parts of the indexer cluster, and the indexer services that it creates select the indexers
-deployed by all the IndexerCluster parts, while the indexer services created by indexer cluster only select the indexers that it manages.
+### Monitoring Console
+The Monitoring Console provides detailed topology and performance information about your Splunk Enterprise deployment. The Monitoring Console pod is referenced by using the `monitoringConsoleRef` parameter. The Monitoring Console pod will periodically check for the existence of new or existing pods in the namespace, and automatically configure a connection to those pods.
```

Comment:
Need to rephrase "The Monitoring Console pod will periodically check for the existence of new or existing pods in the namespace, and automatically configure a connection to those pods"